### PR TITLE
Pr fixup

### PR DESCRIPTION
**Pull request by jpatton-USGS**
This PR cleans up a number of minor issues with the code style, updates the build system to unzip the large slab models, and updates the regression unit tests to be more comprehensive

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,7 @@ build
 lib
 bin
 
+# slab files in zip, don't want to readd them
```
Comment (mguy-usgs) [inline .gitignore:12]: totally minor, but I don't understand this comment, it says slab files in zip, but the extension is txt and then what do you mean by "don't want to read them" ??

File: build.gradle [hunk 2]
```diff
@@ -168,6 +166,15 @@ task cloneExternalProjects {
   }
 }
 
+// unzip large model files
+task unzipModelFiles(type: Copy) {
+    def zipFile = file('models/slabmaster.zip')
```
Comment (mguy-usgs) [inline build.gradle:171]: should these be hard coded? seems other places in the code might want to find them?

File: src/main/java/gov/usgs/locator/LocService.java [hunk 3]
```diff
@@ -135,21 +130,21 @@ public LocOutput getLocation(final LocInput in) throws LocationException {
 
     // print input for debugging
     LOGGER.info("Input: \n" + event.getHydraInput(false));
-    
+
     // make sure we have a slab resolution
-    if(slabRes == null) {
-    	slabRes = "2spd";
+    if (in.SlabResolution == null) {
+      in.SlabResolution = "2spd";
```
Comment (mguy-usgs) [inline src/main/java/gov/usgs/locator/LocService.java:136]: minor - should this be a constant (e.g. DEFAULTSLABRESOLUTION)

File: src/main/java/gov/usgs/locator/Locate.java [hunk 4]
```diff
@@ -245,28 +247,39 @@ public LocStatus doLocation() {
...
       // This should never happen either.
       LOGGER.severe("Illegal tau partial integral");
-      //    e.printStackTrace();
-      //    System.out.println(e.toString());
-      e.printStackTrace();
+
+      StringWriter sw = new StringWriter();
+      e.printStackTrace(new PrintWriter(sw));
+      String exceptionAsString = sw.toString();
+      LOGGER.severe(exceptionAsString);
```
Comment (mguy-usgs) [inline src/main/java/gov/usgs/locator/Locate.java:272]: Thank you for cleaning up the logging! :-)

File: src/main/java/gov/usgs/locaux/AbstractZoneStats.java [hunk 5]
```diff
@@ -1,208 +1,215 @@
...
     return interpolate.interpolateBayesDepth(latitude, longitude, this);
-	}
-  
-	/**
-	 * @return the depth source identification enumeration string
-	 */
+  }
+
+  /**
+   * Function to get depth source type for this ZoneStats
```
Comment (mguy-usgs) [inline src/main/java/gov/usgs/locaux/AbstractZoneStats.java:127]: Thank you for improving code comments!

**Review by mguy-usgs — APPROVED** [review]
Good job with the general format and logging clean up!

File: build.gradle (hunk 2 above)
Comment (jpatton-USGS) [reply build.gradle:171]: this just copies into the right area for the build, it's a variable passed in as part of the command line

File: .gitignore (hunk 1 above)
Comment (jpatton-USGS) [reply .gitignore:12]: I mean, I don't want someone to accidentally do a git add at the top level, and add these files to be committed, after which github will reject the files as too big

File: src/main/java/gov/usgs/locator/LocService.java (hunk 3 above)
Comment (jpatton-USGS) [reply src/main/java/gov/usgs/locator/LocService.java:136]: Probably, but it's only in one place in the repo so it didn't seem worth it.

**Pull request merged**